src/clang_utils.cc

```diff
@@ -57,6 +76,8 @@ optional<lsDiagnostic> BuildAndDisposeDiagnostic(CXDiagnostic diagnostic,
 
   switch (clang_getDiagnosticSeverity(diagnostic)) {
     case CXDiagnostic_Ignored:
+      assert(0);
```

Comment:
Log a warning instead of asserting?

Reply:
LLVM uses `llvm_unreachable()`, i don't know the counterpart in cquery.

Reply:
Is this actually unreachable? Maybe we should just silently ignore the diagnostic if we get CXDiagnostic_Ignored.

Reply:
Changed to `// llvm_unreachable`